scripts/vfx/build_oiio.sh

```diff
@@ -14,7 +14,7 @@ fi
 mkdir build
 cd build
 cmake -DCMAKE_INSTALL_PREFIX=${ASWF_INSTALL_PREFIX} \
-      -DOIIO_BUILD_TOOLS=ON \
+      -DOIIO_BUILD_TOOLS=OFF \
```

Comment:
Any idea why the tools fail to build?

Reply:
looks like the failure is not happening for 2020, I'd rather we do a `if [[ $OIIO_VERSION -le 2.0.8 ]]` and not build tools for older version?

Reply:
Unclear on why that would happen on older versions. I was running everything against 2020 which explains why I didn't see an error. I'll pull down all your recent changes and recheck my package builds on one of the older ones like 2018.